src/assets/css/style.css

```diff
@@ -1,3 +1,7 @@
+@tailwind base;
```

Comment:
If I add the tailwind init in the root, The default menu seems to be affected by the tailwind style as well, which is not what we wanted.

How did you managed to scope the tailwind effect to make browsertests pass?